libpod/adapter/images_remote.go

```diff
@@ -0,0 +1,17 @@
+// +build remoteclient
+
+package adapter
+
+import (
+	"github.com/containers/libpod/libpod"
+)
+
+// Info returns information for the host system and its components
+func (r RemoteRuntime) Images() ([]libpod.InfoData, error) {
```

Comment:
According to the comment, the func should be renamed to `Info()`.

Reply:
fixed